tests/nat/eval/red_teaming_evaluator/test_evaluate.py

```
@@ -0,0 +1,464 @@
+    step = create_intermediate_step(IntermediateStepType.LLM_END, name="llm", output_data=output_data)
+
+    if set_data_to_none:
+        step.payload.data = None
+
+    output = red_teaming_evaluator.extract_output_from_step(step)
+
+    assert output == expected_output
+
+
+@pytest.mark.asyncio
+async def test_evaluate_single_output_success(red_teaming_evaluator, mock_llm):
```

Ruff (0.14.8)

Unused function argument: `mock_llm`

(ARG001)